src/FsCheck/Runner.fs

```diff
@@ -120,6 +122,42 @@ module Runner =
                     yield EndShrink result
               yield! test newSize resize rnd1 gen
         }
+        
+#if PCL // sorry windows phone no parallelism for you
+    let private parallelTest limit = test
+#else
+    let internal parallelTest limit initSize resize rnd0 gen =
+        (initSize, rnd0)
+        |> Seq.unfold (fun (initSize, rnd) ->
+            let rnd1, rnd2 = Random.split rnd
```

Comment:
I see. Also note that in some (most?) cases, the size of the generated value influences both the time it takes to generate it (i.e. an n-ary tree of depth 50 takes longer to generate than one of depth 1) and likely the time to run the test as well. It may be necessary to have a fixed (configurable) degree of parallelism, and then assign tests to threads in a likely better balanced way. E.g. say for 2 threads, instead of:
- thread 1: size 1 -> 25
- thread 2: size 25 -> 50

You'd do something like assign all the even sizes to thread 1 and the odd sizes to thread 2.

It's actually possible that on some compute-bound benchmarks plinq would perform better there, as I understand it has fancy work-stealing algorithms etc.